deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/RegisterManager.java

```diff
@@ -123,6 +125,17 @@
     @SerializedName(CAPABILITIES_TAG)
     protected DeviceCapabilities capabilities = null;
 
+    /**
+     * The Digital Twin model id of the device and module
+     * The value will be null for a non-pnp device.
+     * The value will be null for a pnp device until the device connects and registers with the model id.
+     */
+    private static final String MODEL_ID = "modelId";
+    @Expose(serialize = true, deserialize = true)
+    @SerializedName(MODEL_ID)
+    @Setter
+    @Getter
+    protected String modelId = null;
```

Comment:
The other fields in this class don't seem to be using lombok for generating getters and setters. We should be consistent in our usage - update lombok usage for a  fields/ explicitly generate the getter and setters for this field.
What do you think?